integration-tests/regression.test.js

```diff
@@ -86,23 +86,23 @@ describe('Tram-One', () => {
 		// focus would be removed from the component and put on the body of the page
 
 		// focus on the input
-		userEvent.click(getByLabelText(container, 'New Task Label'));
+		await userEvent.click(getByLabelText(container, 'New Task Label'));
```

Comment:
user-events changed to be a promise API, this might mean we can remove a bunch of the `await waitFor` lines actually!